func_tests/test_client.py

```diff
@@ -73,3 +76,31 @@ def test_batch_change(vinyldns_test_context):
     assert change2.record.ptrdname == 'change-test.vinyldns.'
     assert change2.type == RecordType.PTR
 
+    wait_until_recordset_deleted(vinyldns_test_context.client, r.changes[0].zone_id, r.changes[0].record_set_id)
+    wait_until_recordset_deleted(vinyldns_test_context.client, r.changes[1].zone_id, r.changes[1].record_set_id)
+
+
+def test_batch_change_review_process(vinyldns_test_context):
```

Comment:
@pauljamescleary let me know if this is along the lines of what you were thinking for the testing.

Reply:
This looks good.  Does not need to be exhaustive, but it is good to know that a basic flow works.